
From: BlaisorBlade <blaisorblade_spam@yahoo.it>

In the -mm tree (in this moment) and not in 2.6.7 there is another
console_= device in include/linux/console.h; so I renamed the UML one (it's
static).

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/um/drivers/stdio_console.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff -puN arch/um/drivers/stdio_console.c~rename-uml-console-device arch/um/drivers/stdio_console.c
--- 25/arch/um/drivers/stdio_console.c~rename-uml-console-device	2004-07-26 17:43:51.177011288 -0700
+++ 25-akpm/arch/um/drivers/stdio_console.c	2004-07-26 17:43:51.181010680 -0700
@@ -203,7 +203,7 @@ static void console_write(struct console
 		up(&line->sem);
 }
 
-static struct tty_driver *console_device(struct console *c, int *index)
+static struct tty_driver *um_console_device(struct console *c, int *index)
 {
 	*index = c->index;
 	return console_driver;
@@ -217,7 +217,7 @@ static int console_setup(struct console 
 static struct console stdiocons = {
 	name:		"tty",
 	write:		console_write,
-	device:		console_device,
+	device:		um_console_device,
 	setup:		console_setup,
 	flags:		CON_PRINTBUFFER,
 	index:		-1,
_
